man/BamTallyParam-class.Rd

```diff
@@ -14,7 +14,7 @@
 }
 
 \usage{
-  BamTallyParam(genome, which = GRanges(),
+  BamTallyParam(genome, which = GenomicRanges_OR_GRangesList(),
```

Comment:
I think this is a mistake. There is no such function `GenomicRanges_OR_GRangesList()`.